core/amber/src/main/resources/python_udf/texera_udf_main.py

```diff
@@ -1,65 +1,85 @@
+import contextlib
 import importlib.util
 import logging
 import os
 import sys
 from datetime import datetime
 from pathlib import Path
 
+from loguru import logger
+
 from operators.texera_filter_operator import TexeraFilterOperator
 from operators.texera_map_operator import TexeraMapOperator
 from server.udf_server import UDFServer
 
 
-def init_root_logger(log_input_level: str,
-                     stream_log_level: str, stream_log_fmt: str, stream_datefmt: str,
-                     file_log_dir: str, file_log_level: str, file_log_fmt: str, file_datefmt: str) -> None:
+class InterceptHandler(logging.Handler):
```

Comment:
I am having trouble understanding what these classes do. Can you please add comments to make it clear?

Reply:
added some comments